python/ray/train/tests/test_mosaic_trainer.py

```diff
@@ -0,0 +1,146 @@
+import pytest
+
+# torch libraries
+import torch
+import torch.utils.data
+
+# torchvision libraries
+import torchvision
+from torchvision import transforms, datasets
+
+# Import Ray libraries
+import ray
+from ray.air.config import ScalingConfig
+import ray.train as train
+from ray.air import session
+
+# import ray-mosaic integration libraries
+from ray.train.mosaic import MosaicTrainer
+
+# import composer training libraries
+from torchmetrics.classification.accuracy import Accuracy
+from composer.core.evaluator import Evaluator
+from composer.models.tasks import ComposerClassifier
+import composer.optim
+from composer.loggers import InMemoryLogger
+from composer.algorithms import LabelSmoothing
+
+
+mean = (0.507, 0.487, 0.441)
+std = (0.267, 0.256, 0.276)
+cifar10_transforms = transforms.Compose(
+    [transforms.ToTensor(), transforms.Normalize(mean, std)]
+)
+# data_directory = "./data" ## TODO : remove the following line
+data_directory = "~/Desktop/workspace/data"
```

Comment:
we can use a publicly hosted dataset by us on s3, we have a tar.gz hosted on 

`s3://air-example-data/cifar-10-python.tar.gz`

Reply:
How would this be better than just using `torchvision.datasets.CIFAR10`